lib/api/core/auth/formatProcessing.js

```diff
@@ -19,57 +19,40 @@
  * limitations under the License.
  */
 
-const _ = require('lodash');
-
 module.exports = {
   /**
    * Serializes role and transforms it into a POJO
    *
    * @param {Role} role
    * @returns {object}
    */
-  formatRoleForSerialization: role => {
-    const response = {_id: role._id};
-
-    response._source = _.omit(role, ['_id', 'closures', 'restrictedTo']);
+  serializeRole: role => {
+    const { _id, restrictedTo, ..._source } = role; // eslint-disable-line no-unused-vars
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "restrictedTo" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1854)